FirebaseAppCheck/CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
-# 8.1.0 -- M97
+# 8.1.0 -- M98
 - [added] Apple's App Attest attestation provider support. (#8133)
+- [changed] Token auto-refresh optimizations. (#8232)
```

Comment:
Something doesn't seem right here: 8.1.0 is M97 and 8.2.0 is M98

Reply:
Good catch, I'll correct it with 8.2.0 M98.

Reply:
We didn't include App Attest provider into M97 because we had last minute changes that were not E2E tested. So the updated changelog reflects the actual order of things - we add App Attest provider and optimize token refresh in the M98 milestone release with expected Firebase version 8.2.0.

Reply:
Thanks for clarifying, I understand better!